src/main.jsx

```diff
@@ -46,6 +47,11 @@ const router = createBrowserRouter([
     element: <AssessmentPage />,
     errorElement: <ErrorPage />,
   },
+  {
+    path: "/quiz",
+    element: <QuizPage />,
+    errorElement: <ErrorPage />,
+  },
```

Comment:
Please merge the current main branch into yours, there was a bigger change in the router and i think you have to adapt your code to it